stable/README.md

```diff
@@ -37,37 +40,33 @@ $ mv tiller /usr/local/bin
 If you're running on Mac, the curl commands above would be:
 
 ```bash
-$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.14.1-darwin-amd64.tar.gz | tar xz
+$ curl -s https://storage.googleapis.com/kubernetes-helm/helm-v2.16.1-darwin-amd64.tar.gz | tar xz
 $ cd darwin-amd64
 $ mv helm /usr/local/bin
 $ mv tiller /usr/local/bin
 ```
 
-The following command will configure the current users environment for Helm; it will create a `.helm` directory in `${HOME}`.
-
-```bash
-$ helm init --client-only
-$HELM_HOME has been configured at /home/clusteradmin/.helm.
-Not installing Tiller due to 'client-only' flag having been set
-Happy Helming!
-```
-
-Alternatively, to keep isolated between individuals sharing a login, you can have separate Kube, Helm, and Tiller state (so run these before the above init command):
+To keep isolated between individuals sharing a login, you can have separate Kubernetes, Helm, and Tiller state. Run these before proceeding.
 
 ```bash
 export TILLER_NAMESPACE=nuodb
 export HELM_HOME=`pwd`/.helm
 export KUBECONFIG=`pwd`/.kube/config
 ```
 
-### Install the Tiller Server
-
-We will be creating the Tiller server in the `kube-system` namespace so that it is available to all projects.
+The following commands will configure the current users environment for Helm; it will create a `.helm` directory in `${HOME}`.
 
 ```bash
-oc project kube-system
+$ helm init --client-only
+$HELM_HOME has been configured at /home/clusteradmin/.helm.
+Not installing Tiller due to 'client-only' flag having been set
+Happy Helming!
 ```
 
+## Install the Tiller Server in Kubernetes
```

Comment:
all of the following steps are only required in OpenShift. The install in vanilla k8s is different.

Reply:
If the vanilla k8s install uses helm and tiller locally and ocp only uses tiller in K8s, I can make that change. Please confirm.

Reply:
I've used those 3 commands in rancher without issue, they are probably cover most bases?  Maybe we should just point out that we need a functioning helm installation, this is an example, refer to helm docs for further detail etc

Reply:
Done.